template/Content/FeatherHttpContent.csproj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>featherhttp_template</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="FeatherHttp" Version="0.1.34-alpha.g005ace427a" />
+    <Content Include="Program.cs" />
```

Comment:
Why are these in the csproj?

Reply:
Program.cs and nuget.config are not required in the ItemGroup - although the csproj vscode extension keeps prompting me to add them XD. I've now removed them in my most recent commit 'cleaning up template csproj'

Please let me know if you would like additional changes here

Reply:
I don't see the changes yet

Reply:
Apologies, it seems I removed the RootNamespace but not these files as content items. I've now done so in the most recent commit